textproc/chordii/Portfile

```diff
@@ -0,0 +1,52 @@
+PortSystem          1.0
+
+name                chordii
+version             4.5.1
+revision            0
+categories          textproc
+maintainers         breun.nl:nils openmaintainer
+platforms           darwin
+license             GPL-2
+description         Chordii takes input files containing the lyrics and chords of one or more songs and produces a print (PostScript) version of these songs.
+long_description    Chordii, pronounced chord-ee-ee, is a utility that was \
+                    first created by lazy guitarists who got tired of turning \
+                    pages in the middle of the songs they liked. \
+                    \
+                    Chordii takes one or more input files containing the \
+                    lyrics and chords of one or more songs and produces a \
+                    print (PostScript) version of these songs. \
+                    \
+                    The output has the following characteristics: \
+                    * titles and sub-titles have been centered, \
+                    * the lyrics appear in the font and size of your choice, \
+                    * all chords names appear right above the right lyrics, \
+                    * all chords used in a song appear as grids at the bottom of the page. \
+                    \
+                    Optionally, you can also: \
+                    * generate an index of your songs, \
+                    * have the pages numbered, \
+                    * have chords transposed up or down, \
+                    * print in 2-up or 4-up modes (multiple pages per printed sheet) and, \
+                    * insert tablature and comments. \
+                    \
+                    You have a great many options on the final appearance of your songs.
```

Comment:
According to https://guide.macports.org/chunked/reference.html description is "A one-sentence description of the ported software", so why would this one-sentence `description` be too long? What is the maximum length for a `description`?

I also don't see a maximum length defined for `long_description`. When is a `long_description` too long?